README.md

```diff
@@ -22,19 +22,32 @@ prawn_document do |pdf|
 end
 ```
 
-No need to call `pdf.render`, it is called by `prawn_document`
+No need to call `pdf.render`, it is called by `prawn_document`.
 
-If you want to customize the name of the file should a user try to save it, you can specify the filename in your action:
+You can customize the name of the generated PDF and the file's delivery format by adding a `prawn_options` method to your controller:
+
+```ruby
+def prawn_options
+  {
+    filename: "my-new-document.pdf",
+    disposition: "attachment"
+  }
+end
+```
+
+This uses the [`Content-Disposition` HTTP header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Disposition#As_a_response_header_for_the_main_body). If you've already set this header, `prawn-rails` will *not* override it.
+
+If no options are given, the file's name will default to the name of the called controller action and the delivery format will default to `inline`. You can override the file's name on a per-action basis with `@filename`:
```

Comment:
@westonganger Your [comment](https://github.com/cortiz/prawn-rails/issues/33#issuecomment-412012641) on #33 makes me think you're not a fan of this approach. Would you rather we totally ignore this in the docs?

Reply:
Its ok. Lets leave it for now. I just had the concern that the @filename ivar could somehow be already used for something else.